<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-KKPJ5RQ" failures="1" tests="3" name="Test" time="15.258" errors="0" timestamp="3 Apr 2024 07:06:06 GMT">
  <testcase classname="HomeLyf.test.User" name="userLogin" time="8.539">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
	at HomeLyf.test.User.userLogin(User.java:104)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- userLogin -->
  <testcase classname="HomeLyf.test.User" name="sendEmailOTP" time="0.352"/>
  <testcase classname="HomeLyf.test.User" name="forgot_Pass" time="0.361"/>
</testsuite> <!-- Test -->
